components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -1349,6 +1360,9 @@ interface JsonRpcService {
                       string chain_id) => (array<SPLTokenAmount> balances,
                                            SolanaProviderError error,
                                            string error_message);
+
+  AnkrGetAccountBalance(string account, array<string> chain_ids)
```

Comment:
nit: Should this be `AnkrGetAccountBalances` (plural) as it's fetching multiple balances?
And maybe the `string account` should be `string account_address` for clarity?